packages/test/test-end-to-end-tests/src/test/blobs.spec.ts

```diff
@@ -293,55 +293,53 @@ describeNoCompat("blobs", (getTestObjectProvider) => {
 		assert.strictEqual(snapshot2.stats.treeNodeCount, 1);
 		assert.strictEqual(snapshot1.summary.tree[0].id, snapshot2.summary.tree[0].id);
 	});
-	for (const summarizeProtocolTree of [undefined, true, false]) {
```

Comment:
Do we still need this test for two commit summaries from other services that are still need two commit summaries?

Reply:
Summarization process is used as part of container attachment flow if we have attachment blobs, so likely this test tries to cover this angle. That said, we get proper coverage from using real driver with end-to-end tests, as ODSP vs. AFR behavior would be different. I.e. the pivot here is based on service used, I do not think we need to directly control this process from outside.

Reply:
I want to add that what this test is doing is wrong - if (in the future) ODSP does not support 2-commit summary, then this test will fail 100%. I want to say same about single-commit and let's say memorilicious / R11S, but I guess they work today (even though service does not support single-commit) as driver could simply drop extra sub-tree (though I have not validated that's actually how it works).